scripts/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+case $1 in
```

Comment:
I'm assuming that the main advantage of using a build script like this is doing this conditional environment setup which is harder to do in the makefiles?